getting_started/scripting/gdscript/static_typing.rst

```diff
@@ -387,17 +399,21 @@ Two scripts can’t depend on each other in a cyclic fashion:
 ::
 
     # Player.gd
+
     extends Area2D
     class_name Player
 
+
```

Comment:
The GDScript style guide only requires separating class and function declarations by two blank lines, not the first variable after `class_name`.